notebooks/experiments/conditional_diffusion/dna_diff_baseline_conditional_UNET.ipynb

```diff
@@ -0,0 +1,7119 @@
+{
```

Comment:
<p>Same goes for all class methods except <code>update_average</code>. As they don't return any values we should add <code>None</code> to the method output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinellolab/DNA-Diffusion/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>